### PR TITLE
Set default color transition

### PR DESCRIPTION
**Pull request by xerona**
https://jira.inbcu.com/browse/AUG-822

### CODE REVIEW

File: src/lib/date-range-input/test.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// This file is required by karma.conf.js and loads recursively all the .spec and framework files
```
Comment (craigmichaelmartin) [inline src/lib/date-range-input/test.ts]: Is this file necessary? None of our other components have their own test.ts file.

Comment (xerona) [reply src/lib/date-range-input/test.ts]: I did not notice this. I think i did a bad job cleaning up what i was working on before.
